tex/latex/pgf-pie/tikzlibrarypie.code.tex

```diff
@@ -80,12 +80,14 @@
   \pgfpie@ifx\pgfpie@text\pgfpie@text@inside{%
     % label and number together
     \path (pgfpie@O) -- ++({\pgfpie@midangle}:{\pgfpie@innerpos}) node[align=center]
-    {\pgfpie@scalefont{#3}#4\\\pgfpie@numbertext{#3}};
+    {\pgfpie@scalefont{#3}\pgfpie@labeltext{#4}\\\pgfpie@numbertext{#3}};
```

Comment:
This is beyond my original patch: Perhaps the three occurrences of
```tex
\pgfpie@scalefont{<arg1>}\pgfpie@labeltext{<arg2>}\\\pgfpie@numbertext{<arg1>}
```
can be put into an individual macro, partly to avoid spurious `\\`.

Reply:
I'll leave this for later, since I'd rather have a user-controllable callback for that.  For now I'm going to leave it at this and make a new release.  In the long run I'm going to remove the `scale font` option.